cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -24,16 +24,16 @@
    ========================================================================== #}
 
 {% macro render(options={}) -%}
-    {% set video_url    = options.video.url | default('/') %}
+    {% set video_url = options.video.url | default('/') %}
     {% set image_url = static('img/cfpb_video_cover_card_1380x776.png') %}
     {% set video_id = options.video.id | default('') %}
 
     {# TODO: Move the advanced Jinja logic into Django. #}
-     {% if video_url.find( '?' ) == -1 -%}
+    {% if video_url.find( '?' ) == -1 -%}
         {% set video_url = video_url ~ '?'  %}
     {% endif %}
-    {% if video_url.find( 'autoplay=' ) == -1 -%}
-        {% set video_url = _buildParam( video_url, 'autoplay=1' )  %}
+    {% if video_url.find( 'autoplay=1' ) != -1 -%}
+        {% set video_url = video_url|replace('autoplay=1', '')  %}
```

Comment:
TRIVIAL: You could space out the pipe, as it's done elsewhere in the file: `video_url | replace`

Reply:
Good catch. I'd prefer if we followed the style shown in Jinja's docs, but I'll update it to be consistent with the rest of the file.